peachjam_search/migrations/0015_es_mapping_add_division_action.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 4.2.15 on 2025-04-07 14:14
+import os
+
+from django.db import migrations
+
+
+def forwards(apps, schema_editor):
+    from django.conf import settings
+    from django_elasticsearch_dsl.registries import registry
+
+    if not settings.DEBUG and os.environ.get("ELASTICSEARCH_HOST"):
+        for ix in registry.get_indices():
+            if not ix._mapping:
+                continue
+            print(f"Adding case action and court division mapping for {ix._name}")
+            ix.connection.indices.put_mapping(
+                index=ix._name,
+                body={
+                    "properties": {
+                        "case_action": ix._mapping["case_action"].to_dict(),
+                        "division": ix._mapping["division"].to_dict(),
+                    }
+                },
+            )
```

Comment:
@longhotsummer i've added this. It updates the mapping but i still get an error in search because the individual documents don't have the fields updated. Do we still have to update the indexed documents?

![image](https://github.com/user-attachments/assets/a62a268e-dfa7-4fa8-9e5e-8a31341c386d)

Reply:
Backfilling is complicated in ES. Adjust the field on the serializer - you can use `allow_null=True`, provide a default, or set `required=False` and it should skip missing attributes.